src/avg.js

```diff
@@ -10,14 +10,23 @@ let sum = require('./sum');
  * @returns {number} The average
  */
 function avg(array) {
+  let count=0;
```

Comment:
The idea is to use the `sum` function you defined in `sum.js`. The code already loads it with `require`.